test/services/measure.service.test.js

```diff
@@ -30,13 +29,20 @@ describe('measure.service', () => {
   beforeAll(async () => {
     const config = buildConfig();
     server = initialize(config);
-    await testSetup(testMeasure, testPatient, testLibrary);
-    await createTestResource(testPatient2, 'Patient');
-    await createTestResource(testGroup, 'Group');
-    await createTestResource(testMeasure2, 'Measure');
-    await createTestResource(deleteMeasure, 'Measure');
-    await createTestResource(testOrganization, 'Organization');
-    await createTestResource(testOrganization2, 'Organization');
+    const dataToImport = [
+      testMeasure,
+      testMeasure2,
+      testPatient,
+      testPatient2,
+      testLibrary,
+      testGroup,
+      testOrganization,
+      testOrganization2,
+      testParam,
+      testParamInvalidResourceType
```

Comment:
`testParam`, `testParamInvalidResourceType` are not resources that should be added to the database as they are used as parameters for tests. 

`deleteMeasure` should be added to this list since it is a measure that is intended to be deleted. Which actually the test that uses it should be updated to ensure the Measure Resource was removed.